Runtimes/Core/cmake/modules/Plist.cmake

```diff
@@ -0,0 +1,32 @@
+function(generate_plist project_name project_version target)
+  if(NOT CMAKE_SYSTEM_NAME STREQUAL "Darwin")
```

Comment:
Windows also uses plists in the toolchain. If we are adding this as a generic module, I think that Windows should also be supported.

Reply:
updated to generate for all platforms but only embedd the plist on Apple platforms